EddieHubCommunity/support
# feat: create Docsify site

#### What does this PR do?

This pull takes the existing markdown documentation and creates a Docsify-powered web view for the content.

#### Description of the task to be completed?

We'll need to approve+merge this, then Eddie will need to deploy it on GH Pages so we have the live view.

#### How can this be manually tested?

Pull down these changes, then run `npm install` and `npm start`. The page should automatically open in your browser.

#### Any background context you want to provide?

Discussed briefly in the maintainers channel on Discord.

#### Is there any relevant issue to this PR?

Nah.

#### Screenshots (if appropriate)

Yes actually!

![image](https://user-images.githubusercontent.com/63889819/142970396-79f37dbe-4c68-493c-a9c7-c2a3c461d4ca.png)

#### Questions

@eddiejaoude we can pair to deploy the site on GH Pages, or I can send you a screenshot of my config for my Docsify sites. Whichever works for you.
